### ISSUE TITLE
data structure

### ISSUE THREAD

**Issue by emrekeles-arch** [prithuls/MV-SWIN-T#2]
How should the data structure be for training ?

**Comment by prithuls** [comment]
Thanks for your interest in our research.

For `torch.utils.data.DataLoader`, the parameter is a function that returns image_cc, image_mlo, and label in that order. For image_cc and image_mlo, we used `PIL.Image`.  

Hope this helps.
